Final/animation-part1.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Dino Game</title>

```

Comment:
Please create a new div at the end and add your name as `<span> Your name </span>`
Something like this `<div> <span> Your name </span>  </div>`